app/src/main/res/layout/home_fragment.xml

```diff
@@ -10,19 +11,24 @@
       type="org.oppia.app.home.UserAppHistoryViewModel"/>
   </data>
 
-  <androidx.constraintlayout.widget.ConstraintLayout
+  <LinearLayout
+    android:id="@+id/llRoot"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="#F0FFFF">
 
-    <TextView
-      android:id="@+id/welcome_text_view"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:text="@{viewModel.userAppHistoryLiveData.getAlreadyOpenedApp() ? R.string.welcome_back_text : R.string.welcome_text}"
-      app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintLeft_toLeftOf="parent"
-      app:layout_constraintRight_toRightOf="parent"
-      app:layout_constraintTop_toTopOf="parent"/>
+<!--    <org.oppia.app.view.TextInputInteractionView-->
+<!--      android:id="@+id/welcome_text_view"-->
+<!--      android:layout_width="match_parent"-->
+<!--      android:layout_height="wrap_content"-->
+<!--      android:text="@{viewModel.userAppHistoryLiveData.getAlreadyOpenedApp() ? R.string.welcome_back_text : R.string.welcome_text}"-->
+<!--      app:layout_constraintBottom_toBottomOf="parent"-->
+<!--      app:layout_constraintLeft_toLeftOf="parent"-->
+<!--      app:layout_constraintRight_toRightOf="parent"-->
+<!--      android:height="40dp"-->
 
-  </androidx.constraintlayout.widget.ConstraintLayout>
+<!--      android:layout_margin="20dp"-->
+<!--      app:layout_constraintTop_toTopOf="parent"/>-->
+
+  </LinearLayout>
```

Comment:
You don't need to make changes to `home_fragment` file for this PR.

Reply:
had reverted the  changes